tests/wpt/mozilla/tests/mozilla/scroll_root.html

```diff
@@ -7,9 +7,6 @@
         <style>
             body {
                 background: green;
-
-                /* FIXME(mrobinson): Remove this workaround when #25559 is fixed. */
-                margin: 0;
```

Comment:
This requires running `./mach update-manifest`

Reply:
Thanks for pointing that out. Fixed.